module/VuFind/src/VuFind/Search/Base/Results.php

```diff
@@ -901,23 +902,39 @@ protected function buildFacetList(array $facetList, ?array $filter = null): arra
             $filter = $this->getParams()->getFacetConfig();
         }
 
-        // Start building the facet list:
-        $result = [];
-
         $options = $this->getOptions();
-        $translatedFacets = $options->getTranslatedFacets();
         $hierarchicalFacets
             = is_callable([$options, 'getHierarchicalFacets'])
             ? $options->getHierarchicalFacets()
             : [];
-        $hierarchicalFacetSortSettings
-            = is_callable([$options, 'getHierarchicalFacetSortSettings'])
-            ? $options->getHierarchicalFacetSortSettings()
-            : [];
+
+        $result = $this->initializeFacetResults($facetList, $filter, $options);
+
+        $this->addExcludeFilters($result, $filter, $hierarchicalFacets);
+
+        $this->setDisplayTextForFacetValues($result, $hierarchicalFacets, $options);
+
+        $this->buildHierarchicalFacets($result, $hierarchicalFacets, $options);
```

Comment:
I think this is more readable without the extra spacing, though feel free to disregard if you disagree.

```suggestion
        $result = $this->initializeFacetResults($facetList, $filter, $options);
        $this->addExcludeFilters($result, $filter, $hierarchicalFacets);
        $this->setDisplayTextForFacetValues($result, $hierarchicalFacets, $options);
        $this->buildHierarchicalFacets($result, $hierarchicalFacets, $options);
```